Crashes seen when running the k8s e2e tests with KDD

Felix log shows:
```
2017-03-08 11:55:28.244 [PANIC][1274] table.go 695: Failed to program iptables, giving up after retries error=exit status 2 ipVersion=0x4 table="filter"
panic: (*logrus.Entry) (0x1725e00,0xc420441680)

goroutine 47 [running]:
panic(0x1725e00, 0xc420441680)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/projectcalico/felix/vendor/github.com/Sirupsen/logrus.Entry.log(0xc420012640, 0xc420117a10, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc4205c1800, ...)
        /go/src/github.com/projectcalico/felix/vendor/github.com/Sirupsen/logrus/entry.go:124 +0x423
github.com/projectcalico/felix/vendor/github.com/Sirupsen/logrus.(*Entry).Panic(0xc420441630, 0xc4205c1978, 0x1, 0x1)
        /go/src/github.com/projectcalico/felix/vendor/github.com/Sirupsen/logrus/entry.go:169 +0x124
github.com/projectcalico/felix/iptables.(*Table).Apply(0xc4200f8f00, 0x344db59)
        /go/src/github.com/projectcalico/felix/iptables/table.go:695 +0x8a1
github.com/projectcalico/felix/intdataplane.(*InternalDataplane).apply(0xc4203c7400)
        /go/src/github.com/projectcalico/felix/intdataplane/int_dataplane.go:676 +0x21f
github.com/projectcalico/felix/intdataplane.(*InternalDataplane).loopUpdatingDataplane(0xc4203c7400)
        /go/src/github.com/projectcalico/felix/intdataplane/int_dataplane.go:540 +0x73a
created by github.com/projectcalico/felix/intdataplane.(*InternalDataplane).Start
        /go/src/github.com/projectcalico/felix/intdataplane/int_dataplane.go:334 +0x51
2017-03-08 11:55:28.328 [INFO][1314] logutils.go 80: Early screen log level set to debug
2017-03-08 11:55:28.331 [INFO][1314] felix.go 104: Felix starting up buildDate="2017-03-01T17:29:58+0000" gitCommit="1975720524fc9447cec6f76492342e19aa2e14b9" version="2.1.0-rc6"
```

--- a/glide.lock
+++ b/glide.lock
@@ -1,5 +1,5 @@
-hash: cd02b79ca83441a1f9a525b6e5e9f773af15071bc1e11890b7e826919fd23b1d
-updated: 2017-03-01T13:46:09.220570067Z
+hash: 2a26cc5aa99bcb07893e216443e5b898e0082318adf3e2f87da736457904f13a
+updated: 2017-03-08T18:14:30.857330778Z
 imports:
 - name: cloud.google.com/go
   version: 3b1ae45394a234c385be014e9a488f2bb6eef821
@@ -13,7 +13,7 @@ imports:
 - name: github.com/blang/semver
   version: 31b736133b98f26d5e078ec9eb591666edfd091f
 - name: github.com/coreos/etcd
-  version: 17ae440991da3bdb2df4309936dd2074f66ec394
+  version: 7f43fdde7485de5be2922024bb0aed1b14803b20
   subpackages:
   - client
   - pkg/pathutil
@@ -76,7 +76,7 @@ imports:
 - name: github.com/golang/glog
   version: 44145f04b68cf362d9c4df2182967c2275eaefed
 - name: github.com/golang/protobuf
-  version: 69b215d01a5606c843240eab4937eab3acee6530
+  version: c9c7427a2a70d2eb3bafa0ab2dc163e45f143317
   subpackages:
   - proto
 - name: github.com/google/gofuzz
@@ -106,7 +106,7 @@ imports:
 - name: github.com/mipearson/rfw
   version: b66ec87bd85055de3f749a8640e9e4c8053052e4
 - name: github.com/onsi/ginkgo
-  version: bb93381d543b0e5725244abe752214a110791d01
+  version: ab07225d112dc7a93c289ac5b2e12735c2c46035
   subpackages:
   - config
   - extensions/table
@@ -136,7 +136,7 @@ imports:
 - name: github.com/projectcalico/go-yaml-wrapper
   version: 598e54215bee41a19677faa4f0c32acd2a87eb56
 - name: github.com/projectcalico/libcalico-go
-  version: 8de5b8d6e6f0a44208f033e208a2dec986abedb8
+  version: 5fb02394d07d5335f9df9a825e3707d5452a69f3
   subpackages:
   - lib
   - lib/api
@@ -226,7 +226,7 @@ imports:
   - jws
   - jwt
 - name: golang.org/x/sys
-  version: 76cc09b634294339fa19ec41b5f2a0b3932cea8b
+  version: 99f16d856c9836c42d24e7ab64ea72916925fa97
   subpackages:
   - unix
 - name: golang.org/x/text
@@ -383,7 +383,7 @@ imports:
   - transport
 testImports:
 - name: github.com/onsi/gomega
-  version: c463cd2a8578290d4be7a25cba69de81cf35785e
+  version: 1de7ab2df9105aa5c15c4d7e14a8a514e3cb8d4b
   subpackages:
   - format
   - internal/assertion

--- a/glide.yaml
+++ b/glide.yaml
@@ -21,7 +21,7 @@ import:
 - package: github.com/go-ini/ini
   version: ^1.21.0
 - package: github.com/projectcalico/libcalico-go
-  version: v1.1.1
+  version: v1.1.2
   subpackages:
   - lib
 - package: github.com/Sirupsen/logrus

